composer.json

```diff
@@ -25,6 +25,7 @@
         "phpunit/php-file-iterator": "~1.3",
         "phpunit/php-token-stream": "~1.3",
         "phpunit/php-text-template": "~1.2",
+        "phpunit/php-timer": ">=1.0.6",
```

Comment:
I copied it from current [PHPUnit\composer.json](https://github.com/sebastianbergmann/phpunit/blob/5.1/composer.json#L30) from current stable PHPUnit branch, to mantain ubiquity.

What should it be?